systers_portal/templates/account/signup.html

```diff
@@ -0,0 +1,63 @@
+{% extends "account/base.html" %}
+{% load url from future %}
+{% load i18n %}
+{% block head_title %}{% trans "Signup" %}{% endblock %}
+{% block content %}
+<head>
+<style>
+#abc
+{
+  margin-left:350px;
+  font-family: ""Times New Roman", Times, serif;"
+  display: block;
+ 
+
+}
+input {
+    border: 5px solid white; 
+    -webkit-box-shadow: 
+      inset 0 0 8px  rgba(0,0,0,0.1),
+            0 0 15px rgba(0,0,0,0.1); 
+    -moz-box-shadow: 
+      inset 0 0 8px  rgba(0,0,0,0.1),
+            0 0 15px rgba(0,0,0,0.1); 
+    box-shadow: 
+      inset 0 0 8px  rgba(0,0,0,0.1),
+            0 0 15px rgba(0,0,0,0.1); 
+    padding: 9px;
+    background: rgba(255,255,255,0.5);
+    margin: 0 0 7px 0;
+}
+
+</style>
+</head>
+<div id="abc">
+<h1>{% trans "Sign Up" %}</h1><br>
+<form class="signup" id="signup_form" method="post" action="{% url 'account_signup' %}">
+{% csrf_token %}
+<!--{{ form.as_p }}-->
+{% for field in form %}
```

Comment:
Ok. This is one way of doing it, but we already have `django-crispy-forms` and we should use it. I won't comment every line here, since most of the code will be changed once you use crispy-forms. The biggest problem is that this markup is not responsive, the layout looks broken since inputs and the button are different widths, the button height is too small, the spacing is not balanced.

I highly recommend taking a look at `account/login.html`. Pay attention to block names, crispy tag and the whole markup of the page. Try to mimic the looks and adapt it to the situation.